spec/controllers/v1/sessions_controller_spec.rb

```diff
@@ -641,26 +663,6 @@ def expect_logger_msg(level, msg)
         end
       end
 
-      context 'when NoMethodError is encountered elsewhere' do
```

Comment:
Removing this `context` block since we are now adding logic to prevent the NoMethodError exception